pgmpy/independencies/Independencies.py

```diff
@@ -456,7 +464,7 @@ def _return_list_if_str(event):
         If variable is a string returns a list containing variable.
         Else returns variable itself.
         """
-        if isinstance(event, str):
+        if isinstance(event, (str, tuple)):
```

Comment:
@nihil21 This can lead to some unexpected behavior, as the `IndependentAssertion` can accept tuple of variables instead of list as well. In that cases, this would start treating that as a single variable, which would give wrong results. I would instead suggest to make changes in the `DynamicBayesianNetwork` to make sure that if a single variable (tuple) is passed it is always as a list.

Reply:
I agree with you, you're solution is cleaner.